landingzones/tests/test_permissions.py

```diff
@@ -27,10 +30,14 @@
 
 # Local constants
 SHEET_PATH = SHEET_DIR + 'i_small.zip'
+TEST_OBJ_NAME = 'test1.txt'
 
 
 class TestLandingZonePermissionsBase(
-    LandingZoneMixin, SampleSheetIOMixin, TestProjectPermissionBase
+    LandingZoneMixin,
+    SampleSheetIOMixin,
+    TestPermissionMixin,
+    TaskflowAPIPermissionTestBase,
```

Comment:
Since these are the non-taskflow permission tests, this should be based on `TestProjectPermissionBase`. That base view already uses `TestPermissionMixin`, so re-including that is not necessary.